<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroMedicoWindow</class>
 <widget class="QWidget" name="CadastroMedicoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="text">
      <string>Cadastro</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 16pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputLogin">
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputSenha">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Senha</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputNome">
     <property name="placeholderText">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoCadastrar">
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoVoltar">
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>
